lib/phoenix_storybook.ex

```diff
@@ -212,11 +213,14 @@ defmodule PhoenixStorybook do
   end
 
   @doc false
-  defp config_quotes(opts) do
+  defp config_quotes(opts, color_mode_icons) do
     quote do
       @impl PhoenixStorybook.BackendBehaviour
       def config(key, default \\ nil) do
-        Keyword.get(unquote(opts), key, default)
+        case key do
+          :color_mode_icons -> unquote(Macro.escape(color_mode_icons))
+          _ -> Keyword.get(unquote(opts), key, default)
+        end
```

Comment:
Escapes in order to support > 2 elem tuple in config

Reply:
This is overcomplicated, you should undo your changes and add a `Macro.escape(opts)` at line 58